ingestion/src/metadata/ingestion/ometa/auth_provider.py

```diff
@@ -216,18 +216,21 @@ async def auth_token(  # pylint: disable=invalid-overridden-method, too-many-loc
                     "privateKey": self.security_config.privateKey,
                     "clientId": self.security_config.clientId,
                     "token": jwt_token,
-                    "scopes": self.security_config.scopes,
```

Comment:
Instead of removing this line and playing again with the dict below, maybe we can try something like:

`"scopes": self.security_config.scopes or []`

to keep it a bit simpler and being able to read what's going on together

Reply:
I have modified that line, restored it as it was
as it's validated by yaml and pydantic model, it can either be empty brackets or values.
tested the same out as well